README.md

```diff
@@ -94,6 +94,14 @@ Putting it all together, it is 6 variant test cases.
 - Applying the `Web variant` to a `Standard Web project`
 - Applying the `Web variant` to a `Custom Web project`
 
+### Release
```

Comment:
How about we document that the `HEX_API_KEY` secret needs to be set before making a release?

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/63/commits/421a87bde1c932dfd3301dde28b40f7a95a697be @hanam1ni